lldb/test/API/tools/lldb-dap/launch/TestDAP_launch.py

```diff
@@ -16,7 +16,6 @@
 
 
 class TestDAP_launch(lldbdap_testcase.DAPTestCaseBase):
-    @skipIfWindows
```

Comment:
Do we need to check if the windows version is new enough for ConPTY to exist?

Reply:
I added a version check 👍